src/Mod/Arch/ArchCommands.py

```diff
@@ -635,7 +633,7 @@ def removeShape(objs,mark=True):
                     v1 = dims[0].multVec(v1)
                     v2 = dims[0].multVec(v2)
                     line = Draft.makeLine(v1,v2)
-                    wal = ArchWall.makeWall(line,width=width,height=dims[3],name=name)
+                    _ = ArchWall.makeWall(line,width=width,height=dims[3],name=name)
```

Comment:
No actual need to catch the returned value I guess
```suggestion
                    ArchWall.makeWall(line,width=width,height=dims[3],name=name)
```